api/registrations/serializers.py

```diff
@@ -12,13 +12,12 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
                                  help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    embargo_end_date = HideIfRetraction(ser.SerializerMethodField(help_text='When will embargo be lifted?'))
```

Comment:
Please change `help_text` to be more descriptive. @brianjgeiger suggested `When the embargo on this registration will be lifted.`

Also, to keep `help_text` consistent across all serializers in the API, could you please change the three with questions to have statements (`pending_retraction`, `pending_registration_approval`, and `pending_embargo`, which should be re-added.)